src/scripts/safari/SafariPokemonList.ts

```diff
@@ -63,7 +74,7 @@ class SafariPokemonList {
         const endIndex = startIndex + GameConstants.FRIEND_SAFARI_POKEMON;
 
         const pokemon: SafariEncounter[] = shuffledPokemon.slice(startIndex, endIndex).map((p) => {
-            return new SafariEncounter(p, 10, true);
+            return new SafariEncounter(p, 10, [SafariEnvironments.Grass], true);
```

Comment:
We could consider checking if the pokemon is water type or not? But i am not sure if we can make a general rule for that? Should they all only be in water, or should they all only be in grass?
But it's not a problem with this PR, so i will merge this, and then we can make the change in a new PR, if we want to

Reply:
imo pure water (and maybe Water/Ice) should be Water only encounter, Water/Other should be both, everything else grass only

Reply:
That is smart

Reply:
Mentioned on discord that if some pokemon will be water exclusive, we should make sure every safari has water tiles